build-scripts/start_single_course.sh

```diff
@@ -2,7 +2,7 @@
 
 if [ "$1" != "" ]; then
   trap 'kill 0' EXIT
-  hugo server -s site -p 3000 --bind 0.0.0.0 --theme single_course --contentDir $1 &
+  hugo server -s site -p 3000 --bind 0.0.0.0 --theme single_course --contentDir ../$1 &
```

Comment:
This change seems to break single-course serving.

When I ran `start:single_course -- ../site/content/courses/5-111sc-principles-of-chemical-science-fall-2014` I got a blank page at `http://localhost:3000`. Also this warning message from hugo:
```
found no layout file for "HTML" for kind "home": You should create a template file which matches Hugo Layouts Lookup Rules for this combination.

Reply:
PS It would also be helpful to adding a section in the README on running `build:zips` and `start:single_course`

Reply:
@mbertrand Sorry I wasn't more clear on this.  I felt it was awkward to have to prepend the course path with `../` because the command is much easier to use with auto completion.  It should work now if you just use:

`npm run start:single_course -- site/content/courses/18-06sc-linear-algebra-fall-2011/`

It's good that you mention the README though because it should definitely be documented.  I'll get right on that.